[RHACS] [Docs] ROX-30752: Docs for internal CA rotation (#99814)
openshift/openshift-docs: modules/required-user-actions.adoc

// Module included in the following assemblies:
//
// * configuration/internal-certificate-authority-rotation-for-rhacs.adoc

:_mod-docs-content-type: REFERENCE
[id="required-user-actions_{context}"]
= Required user actions

The following table lists the actions you must take for each deployment scenario during certificate authority (CA) rotation or upgrades:

.Required user actions

> [cols="2,2",options="header"]

Reviewer: I would made the second column wider because of the note.

|===
|Deployment scenario |What you must do

|Operator-managed secured clusters

> |No action needed if Central is also Operator-managed. The Operator automates rotation.

Reviewer: ```suggestion
|No action is needed if Central is also Operator-managed. The Operator automates rotation.
```

|Helm-managed secured clusters
|Re-register with a new init bundle or cluster registration secret before the 5-year CA expires.

|Upgrading a Central instance older than 4 years to {rh-rhacs-first} 4.9
a|
[NOTE]
====
Central switches to certificates signed by the new CA and secured clusters older than version 4.9 can not trust the certificates.
====

You can do any of the following tasks:

* Upgrade all secured clusters to {product-title-short} 4.9.

> * Re-register any secured clusters that you can not upgrade.

Reviewer: ```suggestion
* Re-register any secured clusters that you cannot upgrade.
```
